images/linux/scripts/installers/java-tools.sh

```diff
@@ -28,15 +31,21 @@ if isUbuntu20 ; then
     defaultLabel11="(default)"
 fi
 
-apt-get -y install zulu-8-azure-jdk=\*
-apt-get -y install zulu-11-azure-jdk=\*
-apt-get -y install zulu-12-azure-jdk=\*
-update-java-alternatives -s /usr/lib/jvm/zulu-8-azure-amd64
+apt-get -y install adoptopenjdk-8-hotspot=\*
+apt-get -y install adoptopenjdk-11-hotspot=\*
+apt-get -y install adoptopenjdk-12-hotspot=\*
+apt-get -y install adoptopenjdk-13-hotspot=\*
+apt-get -y install adoptopenjdk-14-hotspot=\*
```

Comment:
I don't think we should put 13 and 14 versions on image since they are not LTS
https://github.com/actions/virtual-environments/pull/946#issuecomment-636096092

Reply:
I have removed these versions